python/das_import.py

```diff
@@ -10,8 +10,12 @@ def do_das_query(query):
     Execute das_client for the specified query, and return parsed JSON output
     """
 
-    args = ['dasgoclient', '-json', '-format', 'json', '--query', query]
-    result = subprocess.check_output(args)
+    args = ['dasgoclient', '--format', 'json', '--query', query]
```

Comment:
according to `dasgoclient -h`,
```
  -format string
        Compatibility option with python das_client, use json to get das_client behavior
```
so I would default to `-json` - but the output is not identical: `-json` gives just the `["data"]` part of the `-format json`. The code that uses it is easy to adjust (actually it gets simpler), but the check in https://github.com/cp3-llbb/SAMADhi/pull/31/files#diff-e068c93f5ddd6d675874177f3be1ef45L58 , should become a simple `if len(summary_results) > 1` then, I think.